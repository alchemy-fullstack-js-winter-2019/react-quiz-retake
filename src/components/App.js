import React, { Component } from 'react';
import Controls from './Controls';
import Display from './Display';
import domToImage from 'dom-to-image';
import fileSaver from 'file-saver';

export default class App extends Component {
  state = {
    title: 'Hi',
    subtitle: ''
  };
  
  handleChange = ({ target }) => {
    this.setState({ [target.name]: target.value });
  };
  render() {
    const {
      title, 
      subtitle
    } = this.state;

    return (
      <>
        <Controls
          title={title}
          subtitle={subtitle}
        />
        <Display
          title={title}
          subtitle={subtitle}
        />
      </>
    );
  }
}

export default App;
