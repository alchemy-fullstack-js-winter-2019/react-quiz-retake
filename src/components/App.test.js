import React from 'react';
import renderer from 'react-test-renderer';
import App from '../components/App';

describe('App testing', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
