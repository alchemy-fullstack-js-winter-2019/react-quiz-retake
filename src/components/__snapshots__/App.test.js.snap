// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App matches a snapshot 1`] = `
Array [
  <span>
    1
     / 
    1
  </span>,